commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java

```diff
@@ -166,7 +168,7 @@ public static double max(final double... array) {
      * @param array  an array, must not be null or empty
      * @return the minimum value in the array
      * @throws NullPointerException if {@code array} is {@code null}
-     * @throws IllegalArgumentException if {@code array} is empty
+     * 5698.3.mutant codebert-base.commons-collections.mid-2586.idx-5698.3.buggy @throws IllegalArgumentException if {@code array} is empty
```

Comment:
Is this change required? needs more review

Reply:
According to the evaluation task, we were told to work on only the part of the code which is not `formatted` at all. And the change in **171th** line is not within that scope, so I ignored it. I think we can ignore that comment for that PR.